addons/rose/addon/components/rose/form/radio/group/index.hbs

```diff
@@ -1,3 +1,6 @@
-{{yield (hash
+<div class="radio-field-wrapper {{if @inline "inline"}}">
+  {{yield (hash
   radio=(component 'rose/form/radio/radio' name=@name selectedValue=@selectedValue disabled=@disabled changed=@changed inline=@inline)
-)}}
\ No newline at end of file
+  card=(component 'rose/form/radio/card' name=@name selectedValue=@selectedValue disabled=@disabled changed=@changed inline=@inline layout=@layout)
```

Comment:
Cards are always inline right?

Reply:
you are correct. `inline` isn't needed here